test/control-plane/node/ciliumnodes/v1.20/init.yaml

```diff
@@ -0,0 +1,341 @@
+apiVersion: v1
```

Comment:
Is there a way we could mark those files as auto-generated in GitHub? 

https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github